app/src/main/java/com/example/sharewhatyoucanproject/login/LoginFragment.kt

```diff
@@ -0,0 +1,96 @@
+package com.example.sharewhatyoucanproject.login
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.provider.Settings
+import android.view.LayoutInflater
+import android.view.View
+import android.view.View.GONE
+import android.view.View.VISIBLE
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import com.example.sharewhatyoucanproject.databinding.FragmentLoginBinding
+import com.example.sharewhatyoucanproject.models.UserType
+import com.example.sharewhatyoucanproject.utils.showToast
+
+class LoginFragment : Fragment() {
+
+    private var _binding: FragmentLoginBinding? = null
+    private val binding get() = _binding!!
```

Comment:
make sure to null out the binding in `onDestroyView`

Reply:
done